lib/bootsnap/load_path_cache/core_ext/active_support.rb

```diff
@@ -50,13 +41,21 @@ def remove_constant(const)
           # behaviour.  The gymnastics here are a bit awkward, but it prevents
           # 200+ lines of monkeypatches.
           def load_missing_constant(from_mod, const_name)
-            CoreExt::ActiveSupport.with_bootsnap_fallback(NameError) { super }
+            super
+          rescue NameError => e
+            # NoMethodError is a NameError, but we only want to handle actual
+            # NameError instances.
+            raise unless e.class == NameError
+            raise if from_mod.const_defined?(const_name)
+            without_bootsnap_cache { super }
```

Comment:
Ruby will not know from where this method come. We need to be explicit about who is the caller.

Reply:
I remember fixing that, although I was also developing against the bundle checkout of the gem.  I'm not sure how I missed that in this PR.  Obviously the fix is to prefix it with `CoreExt::ActiveSupport.` like the other calls to it in this file.